diff --git a/node_modules/react-native-reanimated/android/CMakeLists.txt b/node_modules/react-native-reanimated/android/CMakeLists.txt
index d28dc07..1a9e887 100644
--- a/node_modules/react-native-reanimated/android/CMakeLists.txt
+++ b/node_modules/react-native-reanimated/android/CMakeLists.txt
@@ -47,7 +47,8 @@ set(ignoreMe "${JS_RUNTIME_DIR}")
 
 set(BUILD_DIR "${CMAKE_SOURCE_DIR}/build")
 set(ANDROID_CPP_DIR "${CMAKE_SOURCE_DIR}/src/main/cpp")
-set(COMMON_CPP_DIR "${CMAKE_SOURCE_DIR}/../Common/cpp")
+file(COPY "${CMAKE_SOURCE_DIR}/../Common/cpp/" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/Common_cpp")
+set(COMMON_CPP_DIR "${CMAKE_CURRENT_BINARY_DIR}/Common_cpp")
 
 add_subdirectory("${ANDROID_CPP_DIR}/worklets")
 add_subdirectory("${ANDROID_CPP_DIR}/reanimated")
